src/provisioner/cluster.py

```diff
@@ -110,16 +111,9 @@ def _get_storage_kv_list(self, storage_spec: dict, node_id: str):
         """"Return list of confstore key-values for storage."""
         # Create confstore keys,from storage_spec(config from config.yaml).
         storage_kv_list = []
-        key_prefix = f'node>{node_id}>storage'
-        # TODO: Need to remove num cvg_count key later
-        storage_kv_list.append((f'{key_prefix}>cvg_count', len(storage_spec)))
```

Comment:
if we remove cvg_count as part of this PR, then the flow will break right? as lakshita has created two bugs for motr and hare team.

Reply:
Let's not remove it as part of this PR, let's wait for motr and hare fixed then we will remove cvg_count. for now, let's only focus on removing metadata dependency from provisioner code.

Reply:
Removed